cmd/livepeer/livepeer.go

```diff
@@ -357,6 +362,14 @@ func main() {
 		drivers.NodeStorage = drivers.NewS3Driver(br[0], br[1], cr[0], cr[1])
 	}
 
+	if *gsBucket != "" && *gsKey != "" {
```

Comment:
I think we don't need this condition anymore, since the above check returns if either of these variables is empty, right?

Reply:
no, above check only returns if one of the variables empty, and not the other